frontend/src/views/auth/ResetPassword.vue

```diff
@@ -58,7 +58,6 @@
           name="Confirmation"
           vee-id="confirmation"
           vee-rules="pwConfirmed:@password"
-          vee-rules-old="confirmed:password"
```

Comment:
This was just a leftover from an old commit i assume

Reply:
Yes, leftover from #2557